apps/test/unit/AzureTextToSpeechTest.js

```diff
@@ -21,50 +21,6 @@ describe('AzureTextToSpeech', () => {
     playBytesStub.restore();
   });
 
-  describe('enqueueAndPlay', () => {
```

Comment:
All of the changes in this test file are from a previous iteration of this PR, but I kept the change because it makes more sense to unit test `asyncPlayFromQueue_` than to test its behavior via `enqueueAndPlay`